creme/mobile/locale/fr/LC_MESSAGES/django.po

```diff
@@ -27,8 +27,8 @@ msgid ""
 "views.\n"
 "App: Mobile"
 msgstr ""
-"Affiche les utilisateurs qui ont marqué la fiche courante comme favorie dans "
-"les vues mobile.\n"
+"Affiche les utilisateurs qui ont marqué la fiche courante comme favorite "
+"dans les vues mobile.\n"
```

Comment:
https://www.wordreference.com/enfr/favorite
"Favorie" & "Favorite" seems both accepted, but "Favorie" is used is the other places in 'mobile'.